layouts/partials/embedshare.haml

```diff
@@ -0,0 +1,45 @@
+%h3 Embed into your Website
+%input{:type => "text", :size=> "85", :readonly => true, 
+       :value => "<iframe width=\"853\" height=\"480\" src=\"#{oembed_page_url(@item.identifier)}\" frameborder=\"0\" allowfullscreen></iframe>"}
+%h3 Share with others
+%ul
+  %li
+    %a{:id => "btnTweet", :href => twitter_url(@item[:title], page_url(@item)) }
+      via Twitter
+  %li
+    %a{:id => "btnFacebook",
+       :href => facebook_url(@item[:title], page_url(@item)) }
+      via Facebook
+  %li
+    %a{:id => "btnGPlus", :href => googleplus_url(@item[:title], page_url(@item)) }
+      via Google+
+  %li
+    %a{:id => "btnAppNet", :href => appnet_url(@item[:title], page_url(@item)) }
+      via App.net
+  %li
+    %a{:id => "btnMail", :href => mail_url(@item[:title], page_url(@item)) }
+      by Mail
+:javascript
+  $(function () {
+    $('#btnTweet').click(function (e) {
```

Comment:
they would open in the same popup if someone clicks two of the links. You could just use this.id as window-name to come around this.